Gruntfile.js

```diff
@@ -567,6 +567,16 @@ module.exports = function(grunt) {
 				src: [
 					'wp-admin/css/colors/*/*.css'
 				]
+			},
```

Comment:
None of the other default themes are included. Is there a reason to only include these two?